### PR TITLE
Enhance README

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,24 @@ It is strongly recommended to upgrade your PHP version and/or install the missin
 extensions whenever possible. This polyfill should be used only when there is no
 better choice or when portability is a requirement.
 
+Usage
+=====
+
+When using [composer](https://getcomposer.org/) to manage your dependencies, you
+should not `require` the ~~`symfony/polyfill`~~ package, but the standalone ones:
```
Comment (keradus) [inline README.md]: It would be nice to say not only what, but also why.

Comment (nicolas-grekas) [inline README.md]: I just added a sentence about that. OK for you? Any suggestions?

Comment (keradus) [inline README.md]: Cool for me ;) Thanks!
I remember that this suggestion was already mentioned in some issue/pr

File: README.md [hunk 2]
```diff
@@ -32,6 +32,28 @@ It is strongly recommended to upgrade your PHP version and/or install the missin
 extensions whenever possible. This polyfill should be used only when there is no
 better choice or when portability is a requirement.
 
+Usage
+=====
+
+When using [composer](https://getcomposer.org/) to manage your dependencies, you
```
Comment (xabbuh) [inline README.md]: Composer (uppercased C)

File: README.md [hunk 3]
```diff
@@ -32,6 +32,28 @@ It is strongly recommended to upgrade your PHP version and/or install the missin
...
+- `symfony/polyfill-php70` for using the PHP 7.0 functions,
+- `symfony/polyfill-iconv` for using the iconv functions,
+- `symfony/polyfill-intl-grapheme` for using the `grapheme_*` functions,
+- `symfony/polyfill-intl-icu` for using the intl functions and classes,
+- `symfony/polyfill-intl-normalizer` for using the intl normalizer,
+- `symfony/polyfill-mbstring` for using the mbstring functions,
+- `symfony/polyfill-util` for using the polyfill utility helpers,
+- `symfony/polyfill-xml` for using the `utf8_encode/decode` functions.
+
+Requiring ~~`symfony/polyfill`~~ directly would prevent composer from sharing
```
Comment (xabbuh) [inline README.md]: same here

File: README.md [hunk 4]
```diff
@@ -32,6 +32,28 @@ It is strongly recommended to upgrade your PHP version and/or install the missin
...
+correctly polyfills in dependency graphs. As such, it would likelly install
```
Comment (xabbuh) [inline README.md]: likely

Comment (nicolas-grekas) [inline README.md]: fixed

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**